packages/postcss-minify-gradients/src/index.js

```diff
@@ -83,7 +83,7 @@ function optimise(decl) {
           }
 
           if (lastStop && thisStop && isLessThan(lastStop, thisStop)) {
-            arg[2].value = 0;
```

Comment:
This works because the nodes get all serialized anyway, but the node values can't really be literal numbers.